tcms/rpc/tests/test_bug.py

```diff
@@ -0,0 +1,51 @@
+import unittest
+from mock import patch, MagicMock
+
+from django.conf import settings
+
+from tcms.rpc.tests.utils import APITestCase
+
+if 'tcms.bugs.apps.AppConfig' not in settings.INSTALLED_APPS:
+    raise unittest.SkipTest('tcms.bugs is disabled')
+
+
+class TestBug(APITestCase):
+    def _fixture_setup(self):
+        super()._fixture_setup()
+        self.url = "http://some.url"
+        self.expected_result = {
+            'title': 'Bug from cache',
+            'description': 'This bug came from the Django cache'
+        }
+
+    @patch('tcms.rpc.api.bug.tracker_from_url')
+    @unittest.skip('debug')
+    def test_get_details_from_tracker(self, tracker_from_url):
+        returned_tracker = MagicMock()
+        returned_tracker.details.return_value = self.expected_result
+        tracker_from_url.return_value = returned_tracker
+
+        result = self.rpc_client.Bug.details(self.url)
+
+        self.assertEqual(result, self.expected_result)
+        returned_tracker.details.assert_called_once_with(self.url)
+
+    @patch('tcms.rpc.api.bug.cache')
+    @patch('tcms.rpc.api.bug.tracker_from_url')
+    def test_get_details_from_cache(self, tracker_from_url, cache):
+        returned_tracker = MagicMock()
+        tracker_from_url.return_value = returned_tracker
+        cache.get.return_value = self.expected_result
```

Comment:
This one isn't necessary b/c you are trying to verify that caching the bug details & returning them from cache actually works. Here cache.get is a piece of code under test and you don't mock the pieces under test, you mock dependencies/code which are external to the one you are testing.

So before calling the API method just configure the appropriate value in the cache (or use setup), possibly using cache.clean() on teardown, then execute the Bug.details API, compare the returned value and assert that tracker_from_url() wasn't executed.  This is enough to verify that the if condition inside the API function worked as expected.